server/utils/proxyURLs.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable camelcase */
+export const password_change = (org_slug) =>
+  `/api/v1/${org_slug}/account/password/change`;
+export const password_reset = (org_slug) =>
+  `/api/v1/${org_slug}/account/password/reset`;
+export const password_reset_confirm = (org_slug) =>
+  `/api/v1/${org_slug}/account/password/reset/confirm`;
+export const registration = (org_slug) => `/api/v1/${org_slug}/account`;
+export const user_auth_token = (org_slug) =>
+  `/api/v1/${org_slug}/account/token`;
+export const validate_auth_token = (org_slug) =>
+  `/api/v1/${org_slug}/account/token/validate`;
+export const authorize = `/api/v1/authorize`;
+export const user_radius_sessions = (org_slug) =>
+  `/api/v1/${org_slug}/account/session`;
+export const plans = (org_slug) => `/api/v1/${org_slug}/plans`;
+export const create_mobile_phone_token = (org_slug) =>
+  `/api/v1/${org_slug}/account/phone/token`;
+export const verify_mobile_phone_token = (org_slug) =>
+  `/api/v1/${org_slug}/account/phone/verify`;
+export const mobile_phone_number_change = (org_slug) =>
+  `/api/v1/${org_slug}/account/phone/change`;
```

Comment:
There's a lot of repetition here.
What about having a single function that reads an object instead?

Eg:

```javascript
const prefix = '/api/v1/{orgSlug}/';
const paths = {
   password_change: 'account/password/change',
   password_reset: 'account/password/reset',
   // ... etc
}

const reverse = (name, orgSlug) => {
  const path = paths[name];
  if (!path) {
    // throw error
  }
  `${prefix.replace('{orgSlug}', orgSlug)}${path}`;
}
```

You can then import only 1 function (reverse) in each file and call it like `reverse('password_change', orgSlug)`.